.github/workflows/ci.yml

```diff
@@ -69,11 +69,11 @@ jobs:
     - name: Nuget Restore
       run: nuget.exe restore xdp.sln -ConfigFile src/nuget.config
     - name: Build Binary
-      run: msbuild xdp.sln /m  /t:onebranch /p:configuration=${{ matrix.config }} /p:platform=${{ matrix.arch }} /p:SignMode=Off /p:IsAdmin=true /p:BuildStage=Binary
+      run: msbuild xdp.sln /m  /t:onebranch /p:configuration=${{ matrix.config }} /p:platform=${{ matrix.arch }} /p:UndockedOfficial=true /p:BuildStage=Binary
```

Comment:
I assume there's no way to eliminate `/t:onebranch` here with usage of the `UndockedOfficial` parameter?